content/02.introduction.md

```diff
@@ -28,6 +29,7 @@ These technologies are also enabling the production of high-quality de novo geno
 Such technical advances promise to expand the amount of known genomic variation in humans in the near future, and further power SV genotyping studies.
 Representing known structural variation in the wake of increasingly larger datasets poses a considerable challenge, however.
 Standard formats such as VCF are unwieldy when used for SVs due to ambiguity in their specification and limited support across tools.
+[//]: # (I am a bit unsure what is meant by "limited support across tools" here? I would say that VCF is the most supported format when it comes to representing variation, including structural variation. Or am I misunderstanding it?)
```

Comment:
Thanks, will clear this up.  I think we mean that many tools that read VCF don't necessarily support the symbolic SV alleles like <INS>, and almost none support the crazy breakpoint format.